V1/utils/config.py

```diff
@@ -26,18 +26,46 @@ def load_config(path_to_config = './config.json'):
     data = json.loads(data)
     return data
 
+def create_deadline(deadline_value):
+    deadline = 45.0
+    try:
+        if deadline_value < 0:
+            print("WARNING: Value for deadline should not be negative")
+        if deadline_value != 0:
+            deadline = abs(float(deadline_value))
+        else:
+            print("ERROR: Value for deadline cannot be 0. 45.0 will be set instead.")
+    except ValueError as val_err:
+        print("ERROR: Invalid value for deadline. 45.0 will be set instead.")
+        deadline = 45.0
+    return deadline
+
+def create_idle_power(value):
+    idle_power = 5
+    try:
+        if value < 0:
+            print("WARNING: Value for idle_power should not be negative")
+        if value != 0:
+            idle_power = abs(int(value))
+        else:
+            print("ERROR: Value for idle_power cannot be 0. 5 will be set instead.")
```

Comment:
It should raise an exception, not just printing the error. In fact, if there was an error in the "try" block, the "exception" block is triggered.